Pinta.Core/Classes/Document.cs

```diff
@@ -40,13 +40,20 @@ namespace Pinta.Core
 	// somewhat arbitrary.  In general:
 	// Document - Data about the image itself
 	// Workspace - Data about Pinta's state for the image
-	public class Document
+	public class Document : ObservableObject
```

Comment:
It is difficult to remember after this long. 
It might not be necessary anymore. 

Initially the dialog had non-modal instant apply behaviour, which wasn't what I wanted.